buildtools/webpack.commons.js

```diff
@@ -91,13 +91,13 @@ const typeaheadRule = {
   }
 };
 
-const dateFormatterRule = {
-  test: require.resolve('jquery-datetimepicker'),
+/*const dateFormatterRule = {
+  test: require.resolve('../src/misc/php-date-formatter.js'),
   use: {
     loader: 'expose-loader',
     options: 'DateFormatter'
   }
-};
+};*/
```

Comment:
@sbrunner Do you prefer we remove these lines or we keep it what a additional informative comment ?

Reply:
+1 to remove